workflow/snakemake_rules/core.smk

```diff
@@ -22,9 +22,8 @@ rule wrangle_metadata:
         strain_id=lambda w: config.get("strain_id_field", "strain"),
     shell:
         """
-        python3 scripts/wrangle_metadata.py --metadata {input.metadata} \
-                    --strain-id {params.strain_id} \
-                    --output {output.metadata}
+        csvtk -t rename -f strain -n strain_original {input.metadata} \
```

Comment:
Suggestion: explicitly mention the [differences](https://github.com/nextstrain/zika/pull/25#discussion_r1117734453) in the commit message. Otherwise I'd think it's a direct replacement with no functional differences.

Reply:
Agree, will write up the differences and add to commit message.

Reply:
Finished editing commit messages, welcome any suggestions on wording/scope